gflownet/envs/cube.py

```diff
@@ -753,6 +808,8 @@ def _sample_actions_batch_backward(
         # Initialize variables
         n_states = policy_outputs.shape[0]
         is_bts = torch.zeros(n_states, dtype=torch.bool, device=self.device)
+        # Mask of effective dimensions
+        is_effective_dim = ~mask[:, -self.n_dim :]
```

Comment:
This variable is created here but I don't see it used anywhere in this function.

Reply:
Thanks! Fixed in https://github.com/alexhernandezgarcia/gflownet/pull/209/commits/d6303ec7490aa35e74213adfb7346f4b315b39ea